YarnSpinnerTests/Test.cs

```diff
@@ -101,13 +102,26 @@ public void TestIndentation()
 		}
 
 		[Test()]
+		public void TestVariableStorage()
+		{
+			storage.Clear ();
+
+			var path = System.IO.Path.Combine ("TestCases", "VariableStorage.node");
+			dialogue.LoadFile (path);
+
+			foreach (var result in dialogue.Run()) {
+				HandleResult (result);
+			}
+		}
+
+		// [Test()]
```

Comment:
Nope, sorry was just reducing the cases surrounding my own test. Was gonna restore them when the final PR dropped. :)